.github/workflows/test-integration-postgres-mongo-redis.yml

```diff
@@ -80,7 +80,7 @@ on:
 
 jobs:
   test-integration-postgres-mongo-redis:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-latest-16-cores
```

Comment:
@baileyholl Is this sucker perhaps _too_ big?

What is the jump in specs from the previous runners to this?

Reply:
Looks like [GHA Billing](https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions)
```
Operating system | Cores | Per-minute rate (USD)
-- | -- | --
Linux | 2 | $0.008
Linux | 4 | $0.016
Linux | 8 | $0.032
Linux | 16 | $0.064
Linux | 32 | $0.128
Linux | 64 | $0.256
```
So I think we had 2 cores before...?
Memory is proportional to cores, at 4 Gigs each.

Is there a concern beyond cost/ROI, @Wagan8r ?
Jest splintering and overwhelming the database services and bottlenecking itself...?

Reply:
@rbelev Nope, just want to make sure that we're using appropriately sized runners, and that we're actually utilizing the benefit of the spec increase. Also, we've blown past our spending limit before so I want us to walk into this with eyes wide open.

Reply:
And that wasn't just that time when everyone's bill was suddenly millions??

In Account, where this is being used in a feature branch it seems to be shaving ~2 minutes off, 5 down to 3.
I believe Stephen picked this instance size and maybe we set a reminder for ourselves in 2 weeks after deploy to check on the bill and evaluate then?

Reply:
No, I'm referring to the legitimate bill. We've bumped up against it several times.

Just to be clear I'm not at all against this change. Just trying to be a sanity check.

Reply:
This was just to test if a bigger runner had a significant speedup in the Account tests. It saved ~2 minutes total, 16 would be massively overkill for every repo, probably even Account.

Can we set this up where we can determine this runner via a flag/config passed by the repos that inherit this action, that way each repo could specify its own size for running these actions?

Reply:
Potentially, we could use an `if`?

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idif

Would need to do more investigation on it though since I'm not sure if you can compare with input parameters.

Reply:
We'd probably have to set up multiple `jobs` in the workflow (one for each runner size), and only execute the one corresponding to the input value:

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idif